config.mk

```diff
@@ -1,31 +1,35 @@
-COMPILER_BASE=
+CONFIG_DIR:=$(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+
+-include $(CONFIG_DIR)../cheetah_config.mk
+
+COMPILER_BASE?=
 CC=$(COMPILER_BASE)clang
 CXX=$(COMPILER_BASE)clang++
 LINK_CC=$(CC)
 LLVM_LINK=$(COMPILER_BASE)llvm-link
 LLVM_CONFIG=$(COMPILER_BASE)llvm-config
 #
-ABI_DEF=-DOPENCILK_ABI
+ABI_DEF?=-DOPENCILK_ABI
 # If use cheetah
-RTS_DIR=../runtime
-RTS_LIB=libopencilk
-RTS_C_PERSONALITY_LIB=libopencilk-personality-c
-RTS_CXX_PERSONALITY_LIB=libopencilk-personality-cpp
-RTS_PEDIGREE_LIB=libopencilk-pedigrees
+RTS_DIR?=../runtime
+RTS_LIB?=libopencilk
+RTS_C_PERSONALITY_LIB?=libopencilk-personality-c
+RTS_CXX_PERSONALITY_LIB?=libopencilk-personality-cpp
+RTS_PEDIGREE_LIB?=libopencilk-pedigrees
 
 # All runtime libraries and associated files will be placed in
 # `/oath/to/cheetah/lib/<target-triple>`, so that the compiler can easily find
 # all of those files using the flag --opencilk-resource-dir=/path/to/cheetah.
-RTS_LIBDIR_NAME=lib/$(shell $(LLVM_CONFIG) --host-target)
-RESOURCE_DIR=$(PWD)
-RTS_LIBDIR=$(RESOURCE_DIR)/$(RTS_LIBDIR_NAME)
-RTS_OPT=-fopencilk --opencilk-resource-dir=$(RESOURCE_DIR)
+RTS_LIBDIR_NAME?=lib/$(shell $(LLVM_CONFIG) --host-target)
+RESOURCE_DIR?=$(CONFIG_DIR)
+RTS_LIBDIR?=$(RESOURCE_DIR)$(RTS_LIBDIR_NAME)
```

Comment:
Without using a separate `cheetah_config.mk` file, I found I needed to add a `/` in this line to get things to build properly on Linux:
```make
RTS_LIBDIR?=$(RESOURCE_DIR)/$(RTS_LIBDIR_NAME)
```

Reply:
Whoops; just updated to rebase and add that slash back in